ui/src/app/Pages/pages.module.ts

```diff
@@ -4,14 +4,18 @@ import { ContactUsComponent } from './contact-us/contact-us.component';
 import { SharedModule } from 'src/shared/shared-module/shared.module';
 import { PageNotFoundComponent } from './page-not-found/page-not-found.component';
 import { RouterModule, Routes } from '@angular/router';
+import { HomePageComponent } from './home-page/home-page.component';
+import { AuthGuard } from '../core/auth.guard';
 
 
 const routes: Routes = [
+    {path:"home", component:HomePageComponent ,canActivate:[AuthGuard]},
```

Comment:
why add authGuard for home Page?
i think it will be visible for all

Reply:
the home page will show the interviews  will make by interviewers not available to any one not login